### PR TITLE
Experimental Welders for Engieborgs

### PR DESCRIPTION
**Pull request by Gangstawurst**
## About The Pull Request

Adds the ability to research and create a experimental welder cyborg upgrade for engieborgs

## Why It's Good For The Game

When cargo sells of all the fuel, engineers usually have access to new welders or experimental ones, whereas engieborgs are left behind having to rebuy the fuel
Adding the exp welder for Engieborgs would remove the hassle and make them more useful
Technode for the upgrade is up for debate

## Changelog
:cl:
add: Added cyborg experimental welder upgrade
tweak: "Experimental tools" technode now contains the upgrade module

/:cl:

Yell at me if I did something wrong, for it is my first time working sucessfully with github and byond code

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -580,7 +580,7 @@
 	id = "exp_tools"
 	display_name = "Experimental Tools"
 	description = "Highly advanced tools."
-	design_ids = list("exwelder", "jawsoflife", "handdrill", "laserscalpel", "mechanicalpinches", "searingtool")
+	design_ids = list("exwelder", "jawsoflife", "handdrill", "laserscalpel", "mechanicalpinches", "searingtool", "borg_upgrade_exwelder")
```
Comment (Karmic-Skink) [inline code/modules/research/techweb/all_nodes.dm:583]: Please add an //NSV13 tag

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 2]
```diff
@@ -697,4 +697,31 @@
 	if (.)
 		var/obj/item/borg/apparatus/beaker/extra/E = locate() in R.module.modules
 		if (E)
-			R.module.remove_module(E, TRUE) 
\ No newline at end of file
+			R.module.remove_module(E, TRUE)
+
+/obj/item/borg/upgrade/exp_welder
```
Comment (Karmic-Skink) [inline code/game/objects/items/robot/robot_upgrades.dm:702]: Please move this to a new NSV13 file eg nsv13/code/game/objects/items/robot/custom_robot_upgrades.dm or something like that.

File: code/game/objects/items/tools/weldingtool.dm [hunk 3]
```diff
@@ -377,4 +377,16 @@
 		nextrefueltick = world.time + 10
 		reagents.add_reagent(/datum/reagent/fuel, 1)
 
+/obj/item/weldingtool/experimental/cyborg
```
Comment (Karmic-Skink) [inline code/game/objects/items/tools/weldingtool.dm:380]: Also move to new file.

File: code/modules/research/designs/mechfabricator_designs.dm [hunk 4]
```diff
@@ -777,6 +777,15 @@
 	construction_time = 120
 	category = list("Cyborg Upgrade Modules")
 
+/datum/design/borg_upgrade_expwelder
```
Comment (Karmic-Skink) [inline code/modules/research/designs/mechfabricator_designs.dm:780]: And again into new file please.

**Comment by Kmc2000** [comment]
We like to keep our own copy of the files so that we can easily update from bee with less hassle, please do move these changes into copies of these files under the nsv13/ folder

**Comment by Cenrus** [comment]
You could probably get away with merging this upstream.3

**Pull request merged**